INFO:werkzeug: * Running on http://0.0.0.0:3111/ (Press CTRL+C to quit)
INFO:app:11/06/2021, 21:24:25 2020 CNCF Annual Report Article retrieved
INFO:werkzeug:127.0.0.1 - - [06/Nov/2021 21:24:25] "GET /1 HTTP/1.1" 200 -
INFO:werkzeug: * Running on http://0.0.0.0:3111/ (Press CTRL+C to quit)
INFO:werkzeug:127.0.0.1 - - [07/Nov/2021 19:58:41] "GET /create HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [07/Nov/2021 19:58:41] "GET /static/css/main.css HTTP/1.1" 304 -
INFO:werkzeug:127.0.0.1 - - [07/Nov/2021 19:58:45] "POST /create HTTP/1.1" 302 -
INFO:werkzeug:127.0.0.1 - - [07/Nov/2021 19:58:45] "GET / HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [07/Nov/2021 19:59:06] "GET /create HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [07/Nov/2021 19:59:09] "POST /create HTTP/1.1" 302 -
INFO:werkzeug:127.0.0.1 - - [07/Nov/2021 19:59:09] "GET / HTTP/1.1" 200 -
INFO:werkzeug: * Running on http://0.0.0.0:3111/ (Press CTRL+C to quit)
INFO:app:11/07/2021, 20:00:29 Article "test" retrieved
INFO:werkzeug:127.0.0.1 - - [07/Nov/2021 20:00:29] "GET /7 HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [07/Nov/2021 20:00:34] "GET /create HTTP/1.1" 200 -
INFO:app:A new article "test" is created
INFO:werkzeug:127.0.0.1 - - [07/Nov/2021 20:00:39] "POST /create HTTP/1.1" 302 -
INFO:werkzeug:127.0.0.1 - - [07/Nov/2021 20:00:39] "GET / HTTP/1.1" 200 -
INFO:werkzeug: * Running on http://0.0.0.0:3111/ (Press CTRL+C to quit)
INFO:werkzeug:127.0.0.1 - - [07/Nov/2021 20:02:49] "GET / HTTP/1.1" 200 -
INFO:app:Metrics request successfull
INFO:werkzeug:127.0.0.1 - - [07/Nov/2021 20:02:54] "GET /metrics HTTP/1.1" 200 -
INFO:werkzeug: * Running on http://0.0.0.0:3111/ (Press CTRL+C to quit)
ERROR:app:Exception on / [GET]
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/flask/app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/lib/python3/dist-packages/flask/app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/lib/python3/dist-packages/flask/app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/lib/python3/dist-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/usr/lib/python3/dist-packages/flask/app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/lib/python3/dist-packages/flask/app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "app.py", line 38, in index
    connection = get_db_connection()
  File "app.py", line 20, in get_db_connection
    post_counter = len(connection)
TypeError: object of type 'sqlite3.Connection' has no len()
INFO:werkzeug:127.0.0.1 - - [07/Nov/2021 20:07:37] "GET / HTTP/1.1" 500 -
INFO:werkzeug: * Running on http://0.0.0.0:3111/ (Press CTRL+C to quit)
INFO:werkzeug:127.0.0.1 - - [07/Nov/2021 20:10:20] "GET / HTTP/1.1" 200 -
INFO:app:11/07/2021, 20:10:22 Article "2020 CNCF Annual Report" retrieved
INFO:werkzeug:127.0.0.1 - - [07/Nov/2021 20:10:22] "GET /1 HTTP/1.1" 200 -
INFO:app:Metrics request successfull
INFO:werkzeug:127.0.0.1 - - [07/Nov/2021 20:10:28] "GET /metrics HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [07/Nov/2021 20:11:06] "GET / HTTP/1.1" 200 -
INFO:app:Metrics request successfull
INFO:werkzeug:127.0.0.1 - - [07/Nov/2021 20:11:11] "GET /metrics HTTP/1.1" 200 -
